docs/user/ppl/functions/datetime.rst

```diff
@@ -951,7 +899,7 @@ Description
 
 Usage: Returns a string value containing string format specifiers based on the input arguments.
 
-Argument type: TYPE, STRING, where TYPE must be one of the following tokens: [DATE, TIME, DATETIME, TIMESTAMP], and
+Argument type: TYPE, STRING, where TYPE must be one of the following tokens: [DATE, TIME, TIMESTAMP, TIMESTAMP], and
```

Comment:
`timestamp` is found twice here.

Reply:
Nice catch, removed in https://github.com/Bit-Quill/opensearch-project-sql/pull/336/commits/1aec4f9ee0ca2e447f3a4b9cdff2169a98c76fcb